vnext/Microsoft.ReactNative/ReactHost/React.h

```diff
@@ -283,6 +283,12 @@ struct ReactOptions {
       bool value) noexcept;
   static bool UseWebDebugger(winrt::Microsoft::ReactNative::IReactPropertyBag const &properties) noexcept;
 
+  //! Should the instance enable the Fabric UI architecture
+  void SetEnableFabric(bool enabled) noexcept;
+  bool EnableFabric() const noexcept;
```

Comment:
naming seems off? Either call them 
- SetEnableFabric/GetEnableFabric, or
- EnableFabric(bool) (setter) and EnableFabric() (getter)